### DISCUSSION TITLE
event emotes should be seperated in emote tab

### DISCUSSION THREAD

**Discussion by RynTurtle** [Chatterino/chatterino2#5627] [Ideas]
just updated to latest nightly and noticed the event emotes are merged within the twitch tab, I think it would be better if there was another section like twitch-event or something containing the 2020/pride emotes etc. Not sure if its possible but worth an ask :) 
![BH7boRi_d](https://github.com/user-attachments/assets/caa22c18-c0dc-49d2-8f0d-d5ac90a0526b)

**Comment by Felanbird** [comment]
If we didn't put them all in a `Twitch` section, I believe it would have looked like this with our migration to the Helix API
![311379069-42d9c339-14d4-4a7d-b054-17175308dde6](https://github.com/user-attachments/assets/13c45008-896b-489c-965a-9f19a0c84b0b)
